src/views/GameDetailView.vue

```diff
@@ -0,0 +1,81 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useGameLibraryStore } from '../stores/GameLibraryStore.js'; 
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
+
+const gameLibraryStore = useGameLibraryStore();
+
+if(!gameLibraryStore.count) {
+  await gameLibraryStore.getGameDetails(route.params.id);
+}
+</script>
+
+<template>
+  <main>
+    <h1>Game Details</h1>
+
+    <section class="game-detail-section">
+      <h2>{{ gameLibraryStore.gameDetails.name }}</h2>
+
+      <img :src="gameLibraryStore.gameDetails.imageUrl" loading="lazy" />
+
+      <p class="game-description" v-html="gameLibraryStore.gameDetails.description"></p>
+
+      <ul class="game-info-list">
+        <li>
+          <b>min players: </b>{{ gameLibraryStore.gameDetails.minPlayers }}
+        </li>
+        <li>
+          <b>max players: </b>{{ gameLibraryStore.gameDetails.maxPlayers }}
+        </li>
+        <li>
+          <b>playing time: </b>{{ gameLibraryStore.gameDetails.playingTime }}
+        </li>
+      </ul>
+
+      <div class="category_container">
+        <span class="category" v-for="(category, index) in gameLibraryStore.gameDetails.categories" :key="index">
+          {{ category }}
+        </span>
+      </div>
+    </section>
+  </main>
+</template>
+
+<style>
+@import '@/assets/base.css';
+
+.game-detail-section {
+  display: block;
+  margin: 10px;
```

Comment:
Can we bump this up to `margin: 30px`? 

I noticed the `H1` has a left margin of 30px so if we upped it here they would be left aligned.

Reply:
My last comment I promise. It was 15px not 30, I'm sorry 😭